### DISCUSSION TITLE
Format .norg ... translation and future proofing

### DISCUSSION THREAD

**Discussion by benfleis** [nvim-neorg/neorg#195] [General]
I just stumbled upon this project, and love the concept, ambitions and energy!

My Q -- imagine I get into this all the way, start keeping my notes, etc., in the format. What are my future proofing options? While I admire that ambition, not totally clear that norg will gain broad support for tools outside of ripgrep :)

Any plans/thoughts for translators going forward? (Outside the [WIP HTML renderer](https://github.com/nvim-neorg/neorg/discussions/168).)

**Comment by max397574** [comment]
someone is working on a parser to allow to export to the [pandoc](https://github.com/jgm/pandoc) format 
[here](https://github.com/Simre1/neorg-haskell-parser)
